### ISSUE TITLE
⌨ Associate role switcher with description

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#11977]
## ⌨ Accessibility Issue

Currently the role switcher on the manager dashboard has a description but the two are not associated. So, users need to continue past the dropdown to hear what it does.

## 🦋 Expected Behaviour

The description should be programmatically associated with its description.

## 📸 Screenshot

![Screenshot 2024-11-14 160604](https://github.com/user-attachments/assets/44a76be5-cc7e-48d6-8f6f-033366346ac2)

## 🙋‍♀️ Proposed Solution

Use `aria-describedby` to associate the description with the dropdown menu.

## ✅ Acceptance Criteria

- [ ] Dropdown menu is programmatically associated to its description

**Comment by vd1992** [comment]
The dropdown doesn't open for me via keyboard
